app/src/main/java/ca/ualbert/cs/tasko/TaskListAdapter.java

```diff
@@ -62,27 +85,66 @@ public TaskListAdapter(Context context, TaskList dmtasks){
      */
     @Override
     public TaskViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
-        View view = inflater.inflate(R.layout.task_view, parent, false);
+        View view = inflater.inflate(R.layout.task_cardview_layout, parent, false);
         TaskViewHolder holder = new TaskViewHolder(view);
 
         return holder;
     }
 
     /**
-     * Binds The appropriate Data to the ViewHolder.
+     * Binds The appropriate Data to the ViewHolder, this includes a TaskTitle, Description, Status,
+     * Minimum Bid, and potentially, a users Bid on a task and a Photo of the Task.
      * @param holder The ViewHolder data will be bound too.
      * @param position The position within the RecyclerView.
      */
+    @SuppressLint("SetTextI18n")
     @Override
     public void onBindViewHolder(TaskViewHolder holder, int position) {
         Task currentTask = tasks.get(position);
 
         holder.taskTitle.setText(currentTask.getTaskName());
         holder.taskDescription.setText(currentTask.getDescription());
-        holder.taskStatus.setText("Status: " + currentTask.getStatus());
-        //Needs more information then I currently have/ dont know how to implement.
-        //holder.taskPhoto.setImageResource();
 
+        // Tries to get the minimum bid on each task if it exists
+        try{
+            BidList bids = dm.getTaskBids(currentTask.getId(), thiscontext);
+            Bid lowbid = bids.getMinBid();
+            if(lowbid != null){
+                String lowbidValue = Float.toString(lowbid.getValue());
+                holder.taskLowestBid.setText("Lowest Bid: " + lowbidValue);
+            }else holder.taskLowestBid.setText("Make the First Bid!");
+        } catch (NoInternetException e){
+            Toast t = new Toast(thiscontext);
```

Comment:
This is not the proper way to make toasts. I found this out the hard way. Should be something like this:
`Toast.makeText(getApplicationContext(),"Text", Toast.LENGTH_SHORT).show();`